libs/blocks/merch/merch.js

```diff
@@ -560,6 +560,23 @@ export async function initService(force = false, attributes = {}) {
     fetchCheckoutLinkConfigs.promise = undefined;
   }
   const { commerce, env: miloEnv, locale: miloLocale } = getConfig();
+
+  const extraAttrs = [
+    'checkout-workflow-step',
+    'force-tax-exclusive',
+    'checkout-client-id',
+    'allow-override',
```

Comment:
`allow-override` is not controlled by the consumer but by merch.js, and it is set only for non prod domains.
we should remove it from the list.